src/app/core/utils/math/Rect.ts

```diff
@@ -135,7 +135,7 @@ export class Rect {
             Rect.From({ left: rect.right, right: this.right, top: this.top,    bottom: rect.top    }),
             Rect.From({ left: rect.right, right: this.right, top: rect.top,    bottom: rect.bottom }),
             Rect.From({ left: rect.right, right: this.right, top: rect.bottom, bottom: this.bottom }),
-        ].filter(r => r.width > 0 && r.height > 0);
+        ].filter((r) => r.width > 0 && r.height > 0);
```

Comment:
,parenthesis around the filter body, i think the rule of thumb is that if you have a statement that contains a space, i.e. a comparision like ` == ` or ` > ` or ` && ` or something, there should be parenthesis